nix/default.nix

```diff
@@ -5,7 +5,6 @@
   pkgconf,
   makeWrapper,
   cmake,
-  meson,
```

Comment:
what is this troll, I don't think this is a flake bump

Reply:
It's unused 😶‍🌫️

Reply:
oh, forgot about it.